src/debug.js

```diff
@@ -187,4 +187,39 @@ Debug.prototype.getLogger = function (name) {
   return logger;
 };
 
+/**
+ * Create a synchronous 'getLogger' method that binds around an asynchronous
+ * logger by creating a buffer until the asynchronous logger is resolved.
+ * @see {ModuleInternal.loadLinks} for where this method is bound to the created
+ *     external interface.
+ * @method getLoggingShim
+ * @param {Function} asyncMethod The wrapper to 'getLogger'
+ */
+Debug.prototype.getLoggingShim = function (asyncMethod) {
+  return function getLogggerSync(name) {
+    var toResolve = asyncMethod(name),
+      buffer = [],
+      methods = ['debug', 'info', 'log', 'warn', 'error'],
```

Comment:
pulled out as constant. getting it from API definition would be async, so that's harder.